client/src/Dates.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import './Dates.css';
+
+const Dates = () => {
+  const date = new Date();
+  const dayNumber = date.getDate();
+  const weekday = date.getDay();
+  const month = date.getMonth();
+  const daysOfWeek = {
+    0: 'Sun',
+    1: 'Mon',
+    2: 'Tues',
+    3: 'Wed',
+    4: 'Thurs',
+    5: 'Fri',
+    6: 'Sat',
+    7: 'Sun',
+    8: 'Mon',
+    9: 'Tues',
+    10: 'Wed',
+    11: 'Thurs',
+    12: 'Fri',
+    13: 'Sat',
+
+  };
+  const months = {
+    0: 'Jan',
+    1: 'Feb',
+    2: 'Mar',
+    3: 'Apr',
+    4: 'May',
+    5: 'Jun',
+    6: 'Jul',
+    7: 'Aug',
+    8: 'Sep',
+    9: 'Oct',
+    10: 'Nov',
+    11: 'Dec',
+  };
+  const actualMonth = months[month];
+  const actualWeekday = daysOfWeek[weekday];
+  const fourDaysLater = daysOfWeek[weekday + 4];
+  const hour = date.getHours();
+  const minutes = date.getMinutes();
+  return (
+    <div className="expectedDates">
+      <div className="free">
+        <span className="bold">
+          Get it
+          {' '}
```

Comment:
Consider merging this entire span into one or two long {`I am ${something} that is all on one line`} I know airbnbs style makes it feel like you can't put stuff on one line but you can it just has to be either a line of entirely stuff between {} or just a line of strings

Reply:
Sure, sounds good, I'll go ahead and refactor to be on less lines.